textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/source/ScanBasedSourceOperatorTest.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * 
+ */
+package edu.uci.ics.textdb.dataflow.source;
+
+import java.io.IOException;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.common.exception.DataFlowException;
+import edu.uci.ics.textdb.dataflow.constants.LuceneConstants;
+
+/**
+ * @author sandeepreddy602
+ *
+ */
+public class ScanBasedSourceOperatorTest {
+    
+    private SampleDataStore dataStore;
+    private ScanBasedSourceOperator scanBasedSourceOperator;
+    
+    @Before
+    public void setUp() throws Exception{
+        dataStore = new SampleDataStore();
+        dataStore.clearData();
+        dataStore.storeData();
+        scanBasedSourceOperator = new ScanBasedSourceOperator(
+                LuceneConstants.INDEX_DIR, TestConstants.SAMPLE_SCHEMA);
+    }
+    
+    @After
+    public void cleanUp() throws IOException{
+        dataStore.clearData();

```

Comment:
I am clearing the data after running the test, since it is a independent test case and should leave no trails.